src/smart-components/portfolio/portfolio-item-detail/portfolio-item-detail.js

```diff
@@ -77,15 +78,24 @@ const PortfolioItemDetail = () => {
               product={portfolioItem}
               setOpen={setOpen}
               isFetching={isFetching}
-              availability={source.availability_status}
+              availability={availability}
             />
+            {source.notFound && (
+              <Alert
+                className="pf-u-ml-lg pf-u-mr-lg"
+                id="source-unavailable-alert-info"
+                variant="warning"
+                isInline
+                title="Platform for this product is no longer available"
```

Comment:
The title here should match the one in line 98

Reply:
These are actually different cases. This platform is just deleted, but the alert bellow has an existing source but its status is `unavailable`. Susan is working someone on the terminology.